gdal_subsetter/transform.py

```diff
@@ -239,78 +252,34 @@ def process_geotiff(self, source: HarmonySource, basename: str,
 
         return layernames, result
 
-    def process_netcdf(self, source: HarmonySource, basename:str,
-                       input_filename: str, output_dir: str,
-                       layernames: List[str]):
-
-        filelist = []
-
-        variables = (source.process('variables')
-                     or self.get_variables(input_filename))
-
-        for variable in variables:
-            band = None
-            # For non-geotiffs, we refer to variables by appending a file path
-            layer_format = self.read_layer_format(source.collection,
-                                                  input_filename,
-                                                  variable.name)
-            filename = layer_format.format(input_filename)
-
-            layer_id = '__'.join([basename, variable.name])
-
-            # convert a subdataset in the nc file into the GeoTIFF file
-            filename = self.nc2tiff(layer_id, filename, output_dir)
-            if filename:
-                layer_id, filename, output_dir = self.perform_transforms(
-                    layer_id, filename, output_dir, band
-                )
-
-                layernames.append(layer_id)
-
-                filelist.append(filename)
+    def process_zip(
+        self,
+        source: HarmonySource,
+        basename: str,
+        zipfile_name: str,
+        output_dir: str,
+        layernames: list[str],
+    ) -> tuple[list[str], str | None, str]:
+        """Unpack the input zip file and process the individual files it contains.
 
-        result = self.add_to_result(filelist, output_dir)
-
-        if result:
-            process_msg = 'OK'
-        else:
-            process_msg = 'subsets in the nc file are not stackable, not processed.'
-
-        return layernames, result, process_msg
-
-    def process_zip(self, source: HarmonySource, basename: str,
-                    zipfile_name: str, output_dir: str, layernames: List[str]):
-        """ Unpack the input zip file and process the individual files it
-            contains. If there are GeoTIFF files, only these will be
-            processed.
-
-            `HarmonyAdapter.unpack_zipfile` will attempt to stack multiple
-            GeoTIFFs contained within the zip file, and process those as a
-            single file. The possibility of multiple NetCDF-4 files does not
-            seem to be accounted for here.
+        Only GeoTIFF files will be processed. `HarmonyAdapter.unpack_zipfile`
+        will attempt to stack multiple GeoTIFFs contained within the zip file,
+        and process those as a single file.
```

Comment:
Significant change number 2: Only grabbing GeoTIFFs from zips.

That means a tiny bit of clean-up in this method, and then being able to simplify `HarmonyAdapter.unpack_zipfile` and `get_files_from_unzipfiles` (now `get_unzipped_geotiffs`).